test/test_sparql/test_sparql.py

```diff
@@ -957,3 +957,25 @@ def test_sparql_describe(
     subjects = {s for s in r.graph.subjects() if not isinstance(s, BNode)}
     assert subjects == expected_subjects
     assert len(r.graph) == expected_size
+
+
+@pytest.mark.parametrize(
+    "arg, expected_result, expected_valid",
+    [
+        ("abc", "abc", True),
+        ("1234", "1234", True),
+        (r"1234\u0050", "1234P", True),
+        (r"1234\u00e3", "1234\u00e3", True),
+        (r"1234\u00e3\u00e5", "1234ãå", True),
+        (r"1234\u900000e5", "", False),
+        (r"1234\u010000e5", "", False),
+        (r"1234\u001000e5", "1234\U001000e5", True),
+    ],
+)
+def test_expandUnicodeEscapes(arg: str, expected_result: str, expected_valid: bool):
```

Comment:
Not the most ideal to name it differently from the function, but I guess not clearly worse than a `# noqa`:

```suggestion
def test_expand_unicode_escapes(arg: str, expected_result: str, expected_valid: bool):
```